### DISCUSSION TITLE
Create In Memory Cache Table

### DISCUSSION THREAD

**Discussion by EhsanTava** [laravel/framework#47104] [Ideas] [answered]
As you know we can create cache table for our project "php artisan cache:table" , we can just add an option like "--memory" to create that table in memory (with memory engine) . It gonna boost the performance for database caching . and command would be like :
" php artisan cache:table --memory "

**Comment by yuan8** [comment, marked as answer]
so what differents about memcache or redis fetures?

**Reply by henzeb** [reply]
Well, I can think of not having access to such services, nor the power to install them.

**Reply by Sophist-UK** [reply]
The problem is that it is those same hosted services that don't give you access to such services that don't persist memory between http calls.

In the absence of memcache or redis, the options are (I think) file or database cache - and although I have nether benchmarked these myself nor researched other people's benchmarks, I would expect local disk caching to be faster in many cases than a remote database call.
